test/test_jit.py

```diff
@@ -5952,84 +5952,86 @@ def func(a, b):
         self.checkScript(func, inputs, optimize=True)
 
     def test_math_ops(self):
+        def checkMathWrap(func_name, num_args=1, is_float=True, **args):
+            if is_float:
+                checkMath(func_name, num_args, True, **args)
+                checkMath(func_name, num_args, False, **args)
+            else:
+                checkMath(func_name, num_args, is_float, **args)
 
-        def test_floor(x):
-            # type: (float) -> int
-            return math.floor(x)
-
-        def test_ceil(x):
-            # type: (float) -> int
-            return math.ceil(x)
-
-        def test_log_int(x):
-            # type: (int) -> float
-            return math.log(x)
-
-        def test_log_float(x):
-            # type: (float) -> float
-            return math.log(x)
-
-        def test_log_base_float(x, y):
-            # type: (float, float) -> float
-            return math.log(x, y)
-
-        def test_log1p_int(x):
-            # type: (int) -> float
-            return math.log1p(x)
-
-        def test_log1p_float(x):
-            # type: (float) -> float
-            return math.log1p(x)
-
-        def test_log10_int(x):
-            # type: (int) -> float
-            return math.log10(x)
-
-        def test_log10_float(x):
-            # type: (float) -> float
-            return math.log10(x)
+        inf = float("inf")
+        NaN = float("nan")
+        mx_int = 2147483647
```

Comment:
we use `int64_t` internally so we should try be `2 ** 31`, `2 ** 32`, and `2 ** 63` (and the negations)

Reply:
I do the first 2 - I'll add the last one.

Reply:
Actually, I don't think it's a good idea. Python has big ints, so it'll invariably have different behavior than what we do near the boundary.